packages/contact-exchange/src/services/DIDCommService.ts

```diff
@@ -0,0 +1,156 @@
+import { Message, SecretsResolver, Secret } from 'didcomm-node';
+import { v4 as uuidv4 } from 'uuid';
+import { MessageTyp, MessageType } from './DIDCommOOBInvitation';
+import { DidPeerMethod } from '@adorsys-gis/multiple-did-identities/src/did-methods/DidPeerMethod';
+import { PeerDIDResolver } from 'did-resolver-lib';
+
+class DidcommSecretsResolver implements SecretsResolver {
+  private knownSecrets: Secret[];
+
+  constructor(knownSecrets: Secret[]) {
+    this.knownSecrets = knownSecrets;
+  }
+
+  async get_secret(secretId: string): Promise<Secret | null> {
+    return this.knownSecrets.find((secret) => secret.id === secretId) || null;
+  }
+
+  async find_secrets(secretIds: string[]): Promise<string[]> {
+    return secretIds.filter((id) =>
+      this.knownSecrets.some((secret) => secret.id === id),
+    );
+  }
+}
+
+function createSecretsResolver(knownSecrets: Secret[]): DidcommSecretsResolver {
+  return new DidcommSecretsResolver(knownSecrets);
+}
```

Comment:
I do not see the benefit of having this.

Reply:
Not actually.
Thanks for pointing it out